test/e2e/set/scyllacluster/scyllacluster_tls.go

```diff
@@ -260,6 +265,130 @@ var _ = g.Describe("ScyllaCluster", func() {
 		}
 	})
 
+	g.It("should rotate TLS certificate without disrupting ongoing CQL traffic", func() {
+		if !utilfeature.DefaultMutableFeatureGate.Enabled(features.AutomaticTLSCertificates) {
+			g.Skip(fmt.Sprintf("Skipping because %q feature is disabled", features.AutomaticTLSCertificates))
+		}
+
+		ctx, cancel := context.WithTimeout(context.Background(), testTimeout)
+		defer cancel()
+
+		sc := f.GetDefaultScyllaCluster()
+		o.Expect(sc.Spec.Datacenter.Racks).To(o.HaveLen(1))
+		sc.Spec.Datacenter.Racks[0].Members = 1
+
+		framework.By("Creating a ScyllaCluster")
+		sc, err := f.ScyllaClient().ScyllaV1().ScyllaClusters(f.Namespace()).Create(ctx, sc, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for the ScyllaCluster to rollout (RV=%s)", sc.ResourceVersion)
```

Comment:
```suggestion
		framework.By("Waiting for the ScyllaCluster to roll out (RV=%s)", sc.ResourceVersion)
```

Reply:
There're 29 other messages like this one. Leaving as is for consistency.

Reply:
what's the value in making mistakes consistent? We have to start somewhere

Reply:
Feel free to open PR fixing it. This one is not about it.

Reply:
I don't think previous misspellings are a good reason to grow the set, unless that's part of some related parsing.